test/cubits/blocklist/blocklist_test.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hacki/cubits/cubits.dart';
+import 'package:hacki/repositories/preference_repository.dart';
+import 'package:mocktail/mocktail.dart';
+
+class MockPreferenceRepository extends Mock implements PreferenceRepository {}
+
+void main() {
+  final MockPreferenceRepository mockPreferenceRepository =
+      MockPreferenceRepository();
+  late BlocklistCubit blocklistCubit;
+  late List<String> blocklist;
```

Comment:
I would avoid using `late` as much as possible, you can just do `final List<String> blocklist;` here